validation/basic-cube-constraint-ml.ttl

```diff
@@ -1,4 +1,4 @@
-@base <http://example.org/> .
+@base <https://cube.link/shape/basic-cube-constraint-ml#> .
```

Comment:
once with "-ml" ?
@tpluscode

Reply:
I dis not understand your question

Reply:
here is https://cube.link/shape/basic-cube-constraint**-ml**# instead https://cube.link/shape/basic-cube-constraint#

Reply:
The base matches the file name is each case

Reply:
okay, sorry for the noise!